lib/surveyor/models/answer_methods.rb

```diff
@@ -12,10 +12,13 @@ module AnswerMethods
 
       included do
         # Associations
-        belongs_to :question, required: false
+        belongs_to :question, optional: true
         has_many :responses
         has_many :validations, dependent: :destroy
-        attr_accessible *PermittedParams.new.answer_attributes if defined? ActiveModel::MassAssignmentSecurity
+
+        if defined? ActiveModel::MassAssignmentSecurity
```

Comment:
We can remove this (and all other things of this nature) now that we don't support Rails 3.

Reply:
as in... the whole block, or the conditional part?

Reply:
The whole block -- `ActiveModel::MassAsignmentSecurity` was a 3.0 feature that was the precursor to strong parameters. You used to just declare them on the model vs. in a controller (so you literally couldn't update certain model attributes).